.devcontainer/devcontainer.json

```diff
@@ -3,7 +3,7 @@
 {
 	"name": "Python 3",
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "mcr.microsoft.com/devcontainers/python:0-3.11",
+	"image": "mcr.microsoft.com/devcontainers/base:ubuntu-22.04",
```

Comment:
Please revert this back so it doesn't apply to everyone else.

Reply:
yup, thats my bad. I fixed it.